docs/csharp/how-to/index.md

```diff
@@ -0,0 +1,241 @@
+- [Convert a string to a number](../programming-guide/types/how-to-convert-a-string-to-a-number.md).
+- [Use `as` and `is` to safely cast to a different type](../programming-guide/types/how-to-safely-cast-by-using-as-and-is-operators.md).
+- [Define conversion operators for `struct` types](../programming-guide/statements-expressions-operators/how-to-implement-user-defined-conversions-between-structs.md).
+- [Determine if a type is a nullable type](../programming-guide/nullable-types/how-to-identify-a-nullable-type.md).
+- [Convert between nullable and non-nullable value types](../programming-guide/nullable-types/how-to-safely-cast-from-bool-to-bool.md).
+
+## Equality and ordering comparisons
+
+You may create types that define their own rules for equality or define a
+natural ordering among objects of that type.
+
+- [Test for reference based equality](../programming-guide/statements-expressions-operators/how-to-test-for-reference-equality-identity.md).
+- [Define value based equality for a type](../programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type.md).
+
+## Exception handling
+
+Throwing exceptions is how .NET programs report that methods did not
+successfully complete their work. In these articles you'll learn to work
+with exceptions.
+
+- [Handle exceptions using `try` and `catch`](../programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md).
+- [Cleanup resources using `finally` clauses](../programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md).
+- [Recover from non-CLS (Common Language Specification) exceptions](../programming-guide/exceptions/how-to-catch-a-non-cls-exception.md).
+
+## Delegates and events
+
+Delegates and events provide a capability for strategies that involve
+loosely coupled blocks of code.
+
+- [Declare, instantiate, and use delegates](../programming-guide/delegates/how-to-declare-instantiate-and-use-a-delegate.md).
+- [Combine multicast delegates](../programming-guide/delegates/how-to-combine-delegates-multicast-delegates.md).
+
+Events provide a mechanism to publish notifications or subscribe to notifications.
+
+- [Subscribe and unsubscrbe from events](../programming-guide/events/how-to-subscribe-to-and-unsubscribe-from-events.md).
+- [Implement events declared in interfaces](../programming-guide/events/how-to-implement-interface-events.md).
+- [Conform to .NET Framework guidelines when your code publishes events](../programming-guide/events/how-to-publish-events-that-conform-to-net-framework-guidelines.md).
+- [Raise events defined in base classes from derived classes](../programming-guide/events/how-to-raise-base-class-events-in-derived-classes.md).
+- [Store event instances in a dictionary](../programming-guide/events/how-to-use-a-dictionary-to-store-event-instances.md).
+- [Implement custom event accessors](../programming-guide/events/how-to-implement-custom-event-accessors.md).
+
+## LINQ practices
```

Comment:
Yes, there really is too much LINQ content; the LINQ section really overshadows everything else. Most are also not so much language topics as LINQ topics.   I think that I would pare this down to a few of the more basic topics in each category, and possibly plan to create a more thorough list of LINQ how to topics in the LINQ docs later.

Reply:
I pared it down to the first general How-tos. Those focus more on the query syntax, which is closer to the language.